### ISSUE TITLE
Rename BinaryPenaltyReducedFocalXent to be consistent with Keras

### ISSUE THREAD

**Issue by LukeWood** [keras-team/keras-cv#1247]
in https://github.com/keras-team/keras-cv/pull/1241/ we add "BinaryPenaltyReducedFocalXent" as a loss,

In keras all losses have "Loss" postpended on the end of the name.  also if Xent is just CrossEntropy lets avoid an unecessary acronym here -> BinaryPenaltyReducedFocalCrossEntropy
